From 4857afb89c2913af3ae6f7ab3bd9fc139ebd34f8 Mon Sep 17 00:00:00 2001
From: Ankit Jain <ankitja@vmware.com>
Date: Fri, 1 Oct 2021 10:54:05 +0000
Subject: [PATCH] build.xml: Upgraded vuln jar version

- To fix vuln in second level dependency,
  upgraded vuln. jars version.

Signed-off-by: Ankit Jain <ankitja@vmware.com>
---
 build.xml | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/build.xml b/build.xml
index 9c847ff..b91b6eb 100644
--- a/build.xml
+++ b/build.xml
@@ -326,7 +326,7 @@
           <dependency groupId="org.xerial.snappy" artifactId="snappy-java" version="1.1.1.7"/>
           <dependency groupId="net.jpountz.lz4" artifactId="lz4" version="1.3.0"/>
           <dependency groupId="com.ning" artifactId="compress-lzf" version="0.8.4"/>
-          <dependency groupId="com.google.guava" artifactId="guava" version="18.0">
+          <dependency groupId="com.google.guava" artifactId="guava" version="19.0">
             <exclusion groupId="com.google.code.findbugs" artifactId="jsr305" />
             <exclusion groupId="org.codehaus.mojo" artifactId="animal-sniffer-annotations" />
             <exclusion groupId="com.google.guava" artifactId="listenablefuture" />
@@ -337,7 +337,7 @@
           <dependency groupId="org.hdrhistogram" artifactId="HdrHistogram" version="2.1.9"/>
           <dependency groupId="commons-cli" artifactId="commons-cli" version="1.1"/>
           <dependency groupId="commons-codec" artifactId="commons-codec" version="1.9"/>
-          <dependency groupId="commons-io" artifactId="commons-io" version="2.6" scope="test"/>
+          <dependency groupId="commons-io" artifactId="commons-io" version="2.11.0" scope="test"/>
           <dependency groupId="org.apache.commons" artifactId="commons-lang3" version="3.1"/>
           <dependency groupId="org.apache.commons" artifactId="commons-math3" version="3.2"/>
           <dependency groupId="com.googlecode.concurrentlinkedhashmap" artifactId="concurrentlinkedhashmap-lru" version="1.4"/>
@@ -348,9 +348,9 @@
           <dependency groupId="org.antlr" artifactId="antlr-runtime" version="3.5.2">
             <exclusion groupId="org.antlr" artifactId="stringtemplate"/>
           </dependency>
-          <dependency groupId="org.slf4j" artifactId="slf4j-api" version="1.7.7"/>
-          <dependency groupId="org.slf4j" artifactId="log4j-over-slf4j" version="1.7.7"/>
-          <dependency groupId="org.slf4j" artifactId="jcl-over-slf4j" version="1.7.7" />
+          <dependency groupId="org.slf4j" artifactId="slf4j-api" version="1.7.32"/>
+          <dependency groupId="org.slf4j" artifactId="log4j-over-slf4j" version="1.7.32"/>
+          <dependency groupId="org.slf4j" artifactId="jcl-over-slf4j" version="1.7.32" />
           <dependency groupId="ch.qos.logback" artifactId="logback-core" version="1.2.9"/>
           <dependency groupId="ch.qos.logback" artifactId="logback-classic" version="1.2.9"/>
           <dependency groupId="com.fasterxml.jackson.core" artifactId="jackson-core" version="2.12.5"/>
@@ -364,7 +364,7 @@
             <exclusion groupId="junit" artifactId="junit"/>
           </dependency>
           <dependency groupId="org.yaml" artifactId="snakeyaml" version="1.26"/>
-          <dependency groupId="org.apache.thrift" artifactId="libthrift" version="0.9.2">
+          <dependency groupId="org.apache.thrift" artifactId="libthrift" version="0.9.3-1">
 	         <exclusion groupId="commons-logging" artifactId="commons-logging"/>
 	         <exclusion groupId="org.apache.httpcomponents" artifactId="httpclient"/>
 	         <exclusion groupId="org.apache.httpcomponents" artifactId="httpcore"/>
@@ -418,8 +418,8 @@
           <dependency groupId="io.airlift" artifactId="airline" version="0.6">
             <exclusion groupId="com.google.code.findbugs" artifactId="jsr305" />
           </dependency>
-					<dependency groupId="io.netty" artifactId="netty-bom" version="4.1.58.Final" type="pom" scope="provided"/>
-          <dependency groupId="io.netty" artifactId="netty-all" version="4.0.44.Final" />
+					<dependency groupId="io.netty" artifactId="netty-bom" version="4.1.68.Final" type="pom" scope="provided"/>
+          <dependency groupId="io.netty" artifactId="netty-all" version="4.1.68.Final" />
           <dependency groupId="com.google.code.findbugs" artifactId="jsr305" version="2.0.2" scope="provided"/>
           <dependency groupId="com.clearspring.analytics" artifactId="stream" version="2.5.2">
             <exclusion groupId="it.unimi.dsi" artifactId="fastutil" />
-- 
2.23.1

